charts/netdata/templates/daemonset.yaml

```diff
@@ -60,6 +60,9 @@ spec:
         {{- if .Values.child.persistUniqueID }}
         - name: init-nodeuid
           image: netdata/wget
+          resources:
+            requests:
+              cpu: 1m
```

Comment:
Can we set this to something more sensible like 10m or better 50m?

Reply:
it is an init container that does one `wget` call, i think it is not about sensible/not sensible but just to skip k8s scheduler checks - that is why i set it to minimal.

Reply:
made it 10m, ok, this will work for me